nixos/modules/services/misc/paperless.nix

```diff
@@ -252,9 +252,60 @@ in
         '';
       };
     };
+
+    exporter = {
+
+      enable = lib.mkEnableOption "regular automatic document exports";
+
+      directory = lib.mkOption {
+        type = lib.types.str;
+        default = cfg.dataDir + "/exports";
+        defaultText = "\${dataDir}/exports";
+        description = "Directory to store exports.";
+      };
+
+      onCalendar = lib.mkOption {
```

Comment:
Ideally all modules should use the same name for this kind of option.
Let's choose the one that's already used most frequently.
| option name | # modules that use it |
|---|---|
|interval| 35|
|frequency| 11|
|startAt| 8|
|onCalendar| 3|

Reply:
Yeah, there's arguments in all kinds of directions here and I agree that standardizing this should be separate. I'll keep it as is.